pod/perlfunc.pod

```diff
@@ -3633,8 +3633,8 @@ block.  In general, it may not be used to jump into the parameter
 of a binary or list operator, but it may be used to jump into the
 I<first> parameter of a binary operator.  (The C<=>
 assignment operator's "first" operand is its right-hand
-operand.)  It also can't be used to go into a
-construct that is optimized away.
+operand.)  It also can't be used to go into a construct that is optimized
+away.  Finally, it will be made into a fatal exception in Perl 5.42.
```

Comment:
I think this would be better following on from the first sentence of the paragraph:

```
Use of C<goto LABEL> or C<goto EXPR> to jump into a construct is
deprecated and will issue a warning; it will become a fatal error in
Perl 5.42. While still available, it may not be used to go into any construct [...]
```

Reply:
Agreed.  Please check that I've got your wording correct.